data/reference/Instruments.json

```diff
@@ -453,5 +453,26 @@
         "telescope": "JWST",
         "description": "Fixed Slit",
         "reference": null
+    },
+    {
+        "instrument": "IRC",
+        "mode": "Missing",
+        "telescope": "AKARI",
+        "description": null,
+        "reference": null
+    },
+    {
+        "instrument": "IRC",
+        "mode": "NIR",
+        "telescope": "AKARI",
+        "description": null,
+        "reference": null
+    },
```

Comment:
AKARI IRC only needs to be added once.
And use this as a reference: Murakami, H., et al., 2007, The Infrared Astronomical Mission AKARI, PASJ, 59, S369 [[ADS]](http://adsabs.harvard.edu/abs/2007PASJ...59S.369M) [[DOI:10.1093/pasj/59.sp2.S369]](http://dx.doi.org/10.1093/pasj/59.sp2.S369)

Reply:
In what format would suggest for this? Similar to the format for publications (Mura2007) or should I include it as the bibcode?